packages/blade/src/components/Input/TextInput/__tests__/TextInput.web.test.tsx

```diff
@@ -167,6 +167,18 @@ describe('<TextInput />', () => {
     expect(onFocus).toHaveBeenCalledWith({ name, value: userName });
   });
 
+  it('should handle onClick', async () => {
```

Comment:
should probably also add a test for react-native.

Reply:
Noted